src/Kunstmaan/NodeSearchBundle/EventListener/NodeIndexUpdateEventListener.php

```diff
@@ -44,7 +44,9 @@ public function onPostPersist(NodeEvent $event)
     private function index(NodeEvent $event)
     {
         $nodeSearchConfiguration = $this->container->get('kunstmaan_node_search.search_configuration.node');
-        $nodeSearchConfiguration->indexNodeTranslation($event->getNodeTranslation(), true);
+        $nodeTranslation = $event->getNodeTranslation();
+        $nodeSearchConfiguration->indexNodeTranslation($nodeTranslation, true);
+        $nodeSearchConfiguration->indexChildren($event->getNode(), $nodeTranslation->getLang());
```

Comment:
Are we sure we want to do this? It takes over a dozen of seconds to do this on my production machine, not to mention on the dev server and unnecessary load on elasticsearch, etc.